apps/aecore/src/aec_chain_state.erl

```diff
@@ -323,6 +334,30 @@ export_block(#node{type = block, hash = H, content = B}, State) ->
         error -> B
     end.
 
+%%%-------------------------------------------------------------------
+%%% Find missing blocks
+%%%-------------------------------------------------------------------
+
+get_missing_block_hashes(undefined, _, _State) ->
```

Comment:
It looks to me this clause shall be deleted (in this PR or later), and rather assertions shall be added [at the end of the init of the aec_chain server state](https://github.com/aeternity/epoch/blob/3539d7b89b0eb37b3e7cf4cb754cdcc8d6cb7e2b/apps/aecore/src/aec_chain_server.erl#L75) that:
* Field top_header_hash is a binary (and not undefined) - on the basis that chain contains at least genesis block
* Field top_block_hash is a binary (and not undefined) - on the basis that chain contains at least genesis block